AnimationCodelab/finished/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt

```diff
@@ -0,0 +1,702 @@
+/*
+ * Copyright 2021 Google LLC
```

Comment:
The Copyright for all the files should be with the _The Android Open Source Project_
```suggestion
 * Copyright 2021 The Android Open Source Project
```

Reply:
Done.
Replaced in all the files.